judge/migrations/0088_compiler_message_ansi2html.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 2.1.12 on 2019-09-17 13:30
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+import lxml.html as lh
+from lxml.html.clean import clean_html
+
+
+def strip_error_html(apps, schema_editor):
+    Submission = apps.get_model('judge', 'Submission')
+    for sub in Submission.objects.filter(error__isnull=False):
+        sub.error = clean_html(lh.fromstring(sub.error)).text_content() if sub.error else None
+        sub.save(update_fields=['error'])
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('judge', '0087_problem_resource_limits'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='submission',
+            name='error',
+            field=models.TextField(null=True, verbose_name='compile errors'),
+        ),
+        migrations.RunPython(strip_error_html, migrations.RunPython.noop),
```

Comment:
You would use `atomic=True` to run `strip_error_html`  in a transaction if you cared about performance.